libpromises/feature.c

```diff
@@ -0,0 +1,39 @@
+#include <syntax.h>
+#include <string.h>
+#include <cfnet.h>
+#include <sysinfo.h>
+#include <buffer.h>
+
+static const char* features[] = {
+#ifdef HAVE_LIBYAML
+    "yaml",
+#endif
+#ifdef HAVE_LIBXML2
+    "xml2",
```

Comment:
Found the error: This should be ``"xml"``. If you can fix that real quick I don't have to make a new pull request and can run it through the analyzer again.